### ISSUE TITLE
Feature request: encoding process via queue

### ISSUE THREAD

**Issue by arifje** [nystudio107/craft3-transcoder#63]
When encoding multiple gif files to mp4, the template triggers all these ffmpeg processes at the same time. This results in 40 to 50 ffmpeg processes (in our case), which results in a smoking server and ffmpeg errors. 

It would be awesome if we could prevent this, maybe by using the queue for these jobs? Then we could set a delay on these jobs so we create a spread.

**Comment by khalwat** [comment]
Yeah a way around this could be that the parameters needed to do the conversion are stuffed into a queue job, and dolled out to `ffmpeg` as each queue job is run.
